driver/sources/src/main/scala/org/virtuslab/ideprobe/ProbeDriver.scala

```diff
@@ -189,10 +189,12 @@ class ProbeDriver(
   def freezes: Seq[Freeze] = send(Endpoints.Freezes)
 
   /**
-   * Runs the specified test configuration with a test runner containing provided string
-   * or with the first available runner when the string is not present
+   * Runs the specified test configuration with a test runner containing provided runnerNameFragment
+   * or with the first available runner when the runnerNameFragment is not present
    */
-  def run(runConfiguration: TestRunConfiguration): TestsRunResult = send(Endpoints.RunTest, runConfiguration)
+  def run(runConfiguration: TestRunConfiguration, runnerNameFragment: Option[String]): TestsRunResult = {
```

Comment:
it would be better to make 2 overloads with runnerNameFragment: String and without, this method can be private